src/WPF/ArcGISRuntime.WPF.Samples/Samples/Hydrography/AddEncExchangeSet/AddEncExchangeSet.xaml.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright 2017 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using ArcGISRuntime.Samples.Managers;
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Hydrography;
+using Esri.ArcGISRuntime.Mapping;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace ArcGISRuntime.WPF.Samples.AddEncExchangeSet
+{
+    public partial class AddEncExchangeSet
+    {
+        public AddEncExchangeSet()
+        {
+            InitializeComponent();
+
+            // Create the UI, setup the control references and execute initialization
+            Initialize();
+        }
+
+        private async void Initialize()
+        {
+            // Initialize the map with an oceans basemap
+            MyMapView.Map = new Map(Basemap.CreateOceans());
+
+            // Get the path to the ENC Exchange Set
+            string encPath = await GetEncPath();
+
+            // Create the Exchange Set
+            // Note: this constructor takes an array of paths because so that update sets can be loaded alongside base data
+            EncExchangeSet _encExchangeSet = new EncExchangeSet(new string[] { encPath });
+
+            // Wait for the exchange set to load
+            await _encExchangeSet.LoadAsync();
```

Comment:
Rename the variable to 'myEncExchangeSet' as this is not a global/member variable.